integration-tests/local-plugins/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "gatsby-integration-local-plugin",
+  "description": "Gatsby hello world starter",
+  "license": "MIT",
+  "scripts": {
+    "develop": "gatsby develop",
+    "build": "gatsby build",
+    "test": "jest --config=../jest.config.js local-plugins/",
+    "serve": "gatsby serve"
+  },
+  "dependencies": {
+    "gatsby": "^3.0.0-next.0",
```

Comment:
I did not see any docs in the integration test README but to actually test against my WIP changes, I had to use `yarn link`.

If I didn't, I was getting an error with missing node modules 🤔

Reply:
Resolved using `gatsby-dev